frontend/src/app/pages/marker-page/marker-page.component.html

```diff
@@ -1,6 +1,48 @@
 <app-nav-bar>
     <div class="marker-nav">
         <div class="marker-nav--actions">
+            <button mat-mini-fab color="primary" class="window-change-button" (click)="toggleColorWindowPanel()">
+                <mat-icon [svgIcon]="'contrast-box'"></mat-icon>
+            </button>
+            <div *ngIf="colorWindowPanelActive" id="color-window-panel" class="color-window-panel">
+                <mat-card class="color-window-panel--card">
+                    <mat-card-header class="color-window-panel--card__header">
+                        <mat-card-title>Image Window</mat-card-title>
+                        <mat-card-subtitle>Correct image brightness and contrast by tuning parameters below</mat-card-subtitle>
+                        <button mat-icon-button (click)="toggleColorWindowPanel()">
+                            <mat-icon>close</mat-icon>
+                        </button>
+                    </mat-card-header>
+                    <mat-card-content class="color-window-panel--card__content">
+                        <mat-divider [inset]="true"></mat-divider>
+                        <span>
+                            Window width: {{colorWindowWidth}} 
+                        </span>
+                        <mat-slider (change)="changeImageWindowWidth($event.value)"
+                                    [disabled]="false"
+                                    [max]="255"
+                                    [min]="1" 
```

Comment:
Shouldn't it be zero? Window center can be set to zero 😉

Reply:
Window center can be set to zero and it is. Here you are talking about width which is standarized as value greater or equal to 1.